website/docs/cloud-docs/users-teams-organizations/users.mdx

```diff
@@ -18,6 +18,34 @@ User accounts belong to individual people. Each user can be part of one or more
 Use the [Account API](/terraform/cloud-docs/api-docs/account) to get account details, update account information, and change your password.
 
 <!-- BEGIN: TFC:only -->
+## User Session Overview
```

Comment:
I think you might need another line to ensure this header shows up:
```suggestion

## User Session Overview
```

Otherwise it gets chopped off:
<img width="656" alt="Screenshot 2024-01-12 at 3 18 25 PM" src="https://github.com/hashicorp/terraform-docs-common/assets/32436232/85303714-eb6b-43a3-9266-5b529871c89c">